src/app/models/event.ts

```diff
@@ -1,16 +1,11 @@
 export class Event {
-    // constructor(
-    //     public userId: string,
-    //     public userName: string,
-    //     public group: string,
-    //     public assetId: number,
-    //     public assetTitle: string
-    // ) { }
```

Comment:
There's never really  any reason to leave in commented out code. If it's something that was removed and you think you may add back in later then you have VCS.